recommendation/microsoft.owin.json

```diff
@@ -160,6 +160,36 @@
                 "Contains": "public",
                 "NotContains": "override"
               }
+            },
+            {
+              "Name": "AddExpression",
+              "Type": "Class",
+              "Value": "RequestDelegate _next = null; //CTA Change",
+              "Description": "Add a new global variable to replace the OwinMiddleware global abstract variable of Next.",
+              "ActionValidation": {
+                "Contains": "RequestDelegate_next=null;//CTAChange",
```

Comment:
A week or two ago, a suggestion was made to change `// CTA change` to `// Porting Assistant change`. Might want to double check with Mark if that is something we're going to implement

Reply:
Good catch, done.